openhcl/rootfs.config

```diff
@@ -25,16 +25,17 @@ dir /lib/modules 0755 0 0
 # Kernel modules are loaded in sort order; put them in directories appropriately
 # to ensure they are loaded in dependency order.
 
-dir /lib/modules/000    0755 0 0
-dir /lib/modules/001    0755 0 0
-dir /lib/modules/999    0755 0 0
+dir /lib/modules/auto        0755 0 0
```

Comment:
what's the "auto" here for?

Reply:
Anything in auto will be automatically loaded. Modules elsewhere need explicit code to load.

I'll add a comment.